charts/radar-cloudnative-postgresql/values.yaml

```diff
@@ -0,0 +1,130 @@
+# -- Random database password secrets will be created for these users.
```

Comment:
It's good to have the monitoring enabled 
https://github.com/RADAR-base/radar-helm-charts/blob/feature/cloudnativepg/external/cloudnativepg-cluster/values.yaml#L278